qemu/qmp.go

```diff
@@ -755,6 +755,21 @@ func (q *QMP) ExecuteNetdevAdd(ctx context.Context, netdevType, netdevID, ifname
 	return q.executeCommand(ctx, "netdev_add", args, nil)
 }
 
+// ExecuteNetdevAddByFds adds a Net device to a QEMU instance
+// using the netdev_add command by fds and vhostfds. netdevID is the id of the device to add.
+// Must be valid QMP identifier.
+func (q *QMP) ExecuteNetdevAddByFds(ctx context.Context, netdevType, netdevID, tapFds, vhostFds string) error {
```

Comment:
`vhostFds` should be an slice

Reply:
no, it should be a slice of string, since them must match with the fdname (string id) used in `getfd`

Reply:
Yes, I agree with @devimc we need a slice of strings here. This way, the caller doesn't have to know about the specific format expected by QMP here.

Reply:
Updated